recipes/sagemaker-code-editor/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "sagemaker-code-editor" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor.tar.gz  # [linux64]
+  sha256: 3539e0c977ad2230baa60d5f7f87ac35cf77f17f0b63aaaa8b28cc59c573109f  # [linux64]
+  folder: sagemaker-code-editor
+
+build:
+  number: 0
+  skip: true  # [not linux]
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+    - glib         # [linux]
+    - libsecret    # [linux]
+    - libxkbfile   # [linux]
+    - xorg-libx11  # [linux]
+  host:
+    - nodejs 16.*
+    - glib         # [linux]
+    - libsecret    # [linux]
+    - libxkbfile   # [linux]
+    - xorg-libx11  # [linux]
+  run:
+    - nodejs 16.*
+
+test:
+  commands:
+    - sagemaker-code-editor --help
+
+about:
+  home: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor.tar.gz
```

Comment:
This appears to be a tarbal including the binary for the library.

Reply:
Updated to `https://aws.amazon.com/sagemaker/` similar to other sagemaker recipes